mentorship/users/views.py

```diff
@@ -2,23 +2,42 @@
 from rest_framework.views import APIView
 from rest_framework.response import Response
 from rest_framework import status
-from .models import CustomUser
-from .serializers import CustomUserSerializer
+from .models import CustomUser, Skill
+from .serializers import CustomUserSerializer, CustomUserSerializerRead
 
 class UserList(APIView):
+
+    def get_queryset(self):
+        skills = self.request.data['skills']
+        skill_ids = []
+        for skill in skills:
+            skill_ob = Skill.objects.get(name=skill)
+            skill_ids.append((skill_ob.id))
+        return skill_ids
+
     def get(self, request):
         users = CustomUser.objects.all()
-        serializer = CustomUserSerializer(users, many=True)
+        serializer = CustomUserSerializerRead(users, many=True)
         return Response(serializer.data)
 
     def post(self, request):
+        request.data['skills'] = self.get_queryset()
+        print(request.data)
```

Comment:
Was this print left in from testing or is it required for the functionality of the code?

Reply:
Eeeeep - definitely from testing! 🤭 Removing!